### PR TITLE
Feature SilverStripe 4.x compatibility

### PR DESCRIPTION
**Pull request by stephenmcm**
Hi @sheadawson 

This PR upgrades the module to be compatible with SilverStripe4 and changes the namespace/ownership to Symbiote. From my discussions with @nglasl, I understand that it's an easier ownership transition for you to change the owner of the repository and packagist details.

### CODE REVIEW

**Comment by nglasl** [comment]
@stephenmcm, just for consistency with a few other modules we have, could you please change the `symbiote\` in the namespace to upper case `Symbiote\`? @sheadawson, if you provide write access here (and Packagist), we're more than happy to follow the same process as previously done.

- Merge the SS4 changes.
- Delete the fork.
- Change ownership to where the fork was.

**Comment by stephenmcm** [comment]
Thanks @nglasl, swapped over.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Changelog
```
Comment (nglasl) [inline CHANGELOG.md]: @stephenmcm, is there anything that needs adding here for someone that's bumping SS3 to SS4?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      env:
+        - DB=PGSQL
+    - php: 7.1
+      env:
+        - DB=MYSQL
+        - PDO=1
+
+before_script:
+  #
+  # Silverstripe 3
```
Comment (nglasl) [inline .travis.yml]: @stephenmcm, is this going to work with both SS3 and SS4 config together?

File: _config.php [hunk 3]
```diff
@@ -1,12 +1,3 @@
 <?php
```
Comment (nglasl) [inline _config.php]: @stephenmcm, this file can be deleted :)

**Comment by nglasl** [comment]
Sorry @stephenmcm, two last things. Something like this is important when going from SS3 to SS4, otherwise it starts thinking you're using obsolete class names https://github.com/nglasl/silverstripe-extensible-search/blob/master/_config/upgrade.yml and https://raw.githubusercontent.com/nglasl/silverstripe-mediawesome/master/.upgrade.yml.

**Comment by nglasl** [comment]
Let me know when you think this is all good to go, and I can merge for you :)

**Comment by stephenmcm** [comment]
@nglasl this should be all good to merge now. Let me know if you spot anything else.

**Pull request merged**